### DISCUSSION TITLE
debugging autogen studio workflows

### DISCUSSION THREAD

**Discussion by tevslin** [microsoft/autogen#1452] [General]
How can you see the back and forth between the server and the LLM? I'm mystified by some of the decisions group chat manager makes and what would like to look under the covers,

**Comment by cyxyc** [comment]
i dont. i just pray. (and wish there was a way to do it).

**Comment by marklysze** [comment]
You can set the verbose property, I think it's on groupchat or groupchatmanager. It will provide more visibility on the back and forth messages, but not the entire list of messages each time.
